### PR TITLE
feat(studio): add studio integrations secrets to helm chart

### PR DESCRIPTION
**Pull request by JeanBarriere**

### CODE REVIEW

File: templates/studio/studio.deployment.yaml [hunk 1]
```diff
@@ -31,3 +31,25 @@ spec:
...
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Release.Name }}-studio"
+                  key: intercom_app_id
+            - name: INTERCOM_APP_ID
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Release.Name }}-studio"
+                  key: intercom_app_id
+            - name: SENTRY_DSN
```
Comment (williammartin) [inline templates/studio/studio.deployment.yaml]: Should sentry at least be in a different secret as the same credentials are likely to be used by many deployments?

Comment (williammartin) [reply templates/studio/studio.deployment.yaml]: You thumbed up but didn't change it @JeanBarriere ?

Comment (JeanBarriere) [reply templates/studio/studio.deployment.yaml]: I thought I did. Maybe I committed but not pushed. I'll check

Comment (JeanBarriere) [reply templates/studio/studio.deployment.yaml]: I actually changed and pushed. Happy to change again if that's not what you indended

File: templates/studio/studio.secret.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-studio"
+type: Opaque
+data:
+  intercom_app_id: {{ required "A .Values.studio.integrations.intercom is required" .Values.studio.integrations.intercom | b64enc }}
+  fs_org_id: {{ required "A .Values.studio.integrations.fullstory is required" .Values.studio.integrations.fullstory | b64enc }}
+  sentry_dsn: {{ required "A .Values.studio.sentry.dsn is required" .Values.studio.sentry.dsn | b64enc }}
```
Comment (JeanBarriere) [inline templates/studio/studio.secret.yaml]: @williammartin here the integrations are under studio.integrations and sentry is outside of it

Comment (williammartin) [reply templates/studio/studio.secret.yaml:9]: No sorry that's not what I meant. I think in most cases secrets should be bound to the _provider_, not the consumer (except perhaps in cases where there will only be one consumer).

If the runtime, the graphql, the SLS etc etc are all using Sentry, does it make sense for them to look into a secret called `storyscript-studio`?

Comment (JeanBarriere) [reply templates/studio/studio.secret.yaml:9]: @williammartin the DSN is actually unique for the projects, so if those components use sentry, they should get another DSN, no ?

Comment (williammartin) [reply templates/studio/studio.secret.yaml:9]: ahha! I did not know that. Sounds good to me!

**Pull request merged**